code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -206,6 +206,8 @@
 		bust_open()
 
 /obj/structure/closet/attackby(obj/item/W, mob/user, params)
+	if (istype(src, /obj/structure/closet/bsdroppod))
+		return ..()//dont wanna do anything if you're a drop pod

```

Comment:
The proper way to to this is to move the closet stuff into a seperate proc which you override in bsdropped that tells attabyby to return ..()

Reply:
what does this mean